ui/src/templates/reducers/index.ts

```diff
@@ -59,6 +59,17 @@ const templatesReducer = (
         }
         return
       }
+
+      case ActionTypes.RemoveTemplateSummary: {
+        const {templateID} = action.payload
+        const {items} = draftState
+        const deleted = items.filter(l => {
+          return l.id !== templateID
+        })
+
+        draftState.items = deleted
```

Comment:
the name deleted feels a bit misleading, could we set `draftState.items = items.filter .....` directly?

Reply:
I fixed that :)